clang/test/CXX/drs/dr28xx.cpp

```diff
@@ -0,0 +1,41 @@
+// RUN: %clang_cc1 -std=c++20 -verify=expected %s
+
+namespace dr2847 { // dr2847: 19
+
+template<typename>
+void i();
+
+struct A {
+  template<typename>
+  void f() requires true;
+
+  template<>
+  void f<int>() requires true; // expected-error {{explicit specialization cannot have a trailing requires clause unless it declares a function template}}
```

Comment:
As this error only shows up in C++20 mode and higher, directive should read `since-cxx20-error`. You should also convert it to `since-cxx20-error@-1` style, which the rest of DR test suite follows.